docs/demo/kitchen-sink/web-components.rst

```diff
@@ -25,7 +25,54 @@ Cards and tabs provide some extra UI flexibility for your content. Both `sphinx-
 
    This modification is not needed when using the :code:`sphinx-design` extention.
 
-Below you can find some examples of the **cards** and **tabs** created with the :code:`sphinx-design` extention.
+Below you can find some examples of the components created with the :code:`sphinx-design` extension.
+
+Badges and buttons
+------------------
+
+Here are some of the available badges:
+:bdg-primary:`primary`
+:bdg-secondary:`secondary`
+:bdg-success:`success`
+:bdg-primary-line:`primary outline`
+:bdg-secondary-line:`secondary outline`
+:bdg-success-line:`success outline`
+
+Here are some buttons, also using semantic color names. **Note:** in this theme, ``info`` is defined to be the same color as ``primary``, and ``warning`` is the same color as ``secondary``.
+
+.. grid:: auto
+
+    .. grid-item::
+
+        .. button-link:: https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html
```

Comment:
Any reason why all these buttons are pointing to the badge_butons documentation ? I see the point but it's not straight forward when I look at them so I fear it will generate lots of bounce. Maybe simply set the link at the begining and make them point to "#" ?